### ISSUE TITLE
Uncaught TypeError: _this.copyHtml is not a function

### ISSUE THREAD

**Issue by cbk1411** [dundalek/atom-markdown-mindmap#6]
[Enter steps to reproduce below:]
1. Command + Shift + p
2. type "mindmap"
3. select "Copy Html"

**Atom Version**: 1.1.0
**System**: Mac OS X 10.11.1
**Thrown From**: [markdown-mindmap](https://github.com/dundalek/atom-markdown-mindmap) package, v0.2.4
### Stack Trace

Uncaught TypeError: _this.copyHtml is not a function

```
At /Users/will-mb/.atom/packages/markdown-mindmap/lib/main.coffee:49

TypeError: _this.copyHtml is not a function
  at atom-workspace.atom.commands.add.markdown-mindmap:copy-html (/Users/will-mb/.atom/packages/markdown-mindmap/lib/main.coffee:49:10)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:528:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:351:22)
  at HTMLDocument.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:20)

```
### Commands

```
     -5:19.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -4:50.3.0 core:close (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -4:49.5.0 core:save (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -4:48.5.0 core:close (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -4:47.2.0 markdown-preview:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -4:40.9.0 core:close (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:28.9.0 markdown-preview:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -0:25.7.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:23 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:23 markdown-mindmap:toggle (atom-text-editor.editor)
     -0:17.2.0 markdown-preview:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -0:12.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:11 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:01.9.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:01.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.2.0 markdown-mindmap:copy-html (atom-text-editor.editor)
```
### Config

``` json
{
  "core": {
    "themes": [
      "northem-dark-atom-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "atom-html-preview",
      "markdown-scroll-sync"
    ]
  },
  "markdown-mindmap": {}
}
```
### Installed Packages

``` coffee
# User
color-picker, v2.0.13
file-icons, v1.6.11
markdown-mindmap, v0.2.4
markdown-preview-opener, v0.1.1
markdown-table-formatter, v2.5.4
markdown-toc, v0.3.0
markdown-writer, v2.0.1
northem-dark-atom-ui, v0.3.0
svg-preview, v0.6.1
symbols-tree-view, v0.10.1
tree-view-git-status, v0.2.2

# Dev
No dev packages
```

**Comment by dundalek** [comment]
Thanks for the report. I will fix it when #5 is implemented.

**Comment by dundalek** [comment]
Fixed in a new release v0.3.0, `Copy to Html` command is now not listed.
